src/clj/chronograph/handlers/organization.clj

```diff
@@ -0,0 +1,26 @@
+(ns chronograph.handlers.organization
+  (:require [ring.util.response :as response]
+            [chronograph.domain.organization :as organization]
+            [clojure.spec.alpha :as s]))
+
+(defn create
+  "Any authorized user may create a new organization, provided they submit
+  a unique slug. The user who creates an organization is automatically made
+  admin of that organization."
+  [{{:keys [name slug] :as body} :body
+    {:keys [users/id] :as user} :user
+    :as request}]
+  (cond
+    (not user)
```

Comment:
I think this check can be moved into the middleware.

Reply:
In order to move this into a middleware, it would have to specifically wrap the handlers that need authentication. We wouldn't be able to wrap the handler returned by `bidi/make-handler`.

Reply:
We figured we'd do this once a second middleware is added. But, open to doing this now as well.

Reply:
lifted it out into a wrap-authorized-user middleware